### DISCUSSION TITLE
154

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#93] [General]
# 潮流周刊第154期 - 玉鸟集美

封面图来源于周日拍摄，天气特别好，去玉鸟集逛了逛，原相机出图，天是真的蓝，很喜欢这里的建筑。

https://weekly.tw93.fun/posts/154-%E7%8E%89%E9%B8%9F%E9%9B%86%E7%BE%8E

**Comment by VmzXv** [comment]
你好！
本期潮流工具中 Squoosh：好用的图片压缩工具 MUSCLEWIKI：肌肉锻炼网站 与 152 期中推荐是不是重复

**Reply by tw93** [reply]
的确是我疏漏了，已经更新，谢谢反馈

**Reply by VmzXv** [reply]
这周请假了吗？155期呢

**Reply by tw93** [reply]
哈哈哈 对 我在日本玩，回来后实在太不急了，下周一发

**Comment by jiyiren** [comment]
你好，我看你图片引用是 gw.alipayobjects.com 域名，图片是怎么上传上去的呢🧐？

**Reply by tw93** [reply]
这个潮流周刊其实是我在公司内部发的，刚好有一些同学跳出来，问有没有对外版本，由于没有安全问题，所以就迁移出来了

**Comment by Wuli-Blizzard** [comment]
博主好，感谢你分享的高质量内容！请问可不可以做个关键字索引内容哈，想着有些内容好棒回头能够找到😁。当然要看你的时间和精力哈，做了的话就忒方便啦❤️❤️❤️

**Reply by tw93** [reply]
支持(๑•̀ㅂ•́)و✧

**Reply by Wuli-Blizzard** [reply]
好耶ヾ(≧▽≦*)o

**Reply by tw93** [reply]
哈哈哈 支持了，你应该有看到一个 搜索框，在任何地方 按下 / 就可以唤起

**Reply by Wuli-Blizzard** [reply]
哇塞，能看到。博主真的是太棒啦！终于可以搜到我之前看过的其他内容啦嘿嘿嘿！谢谢博主！

**Reply by Wuli-Blizzard** [reply]
太方便啦！

**Reply by tw93** [reply]
哈哈哈 你喜欢就好
谢谢提出这个需求 我做完以后发现很有用

**Reply by Wuli-Blizzard** [reply]
哈哈哈，也得谢谢博主啦！你太赞啦！对读者来说也很有用。希望博主的周刊能得到更多人的喜欢 (*^▽^*) ，
